README.md

```diff
@@ -49,3 +49,12 @@ yakut sub 1001:uavcan.si.unit.angle.Scalar.1.0 --with-metadata
 ---
 ...
 ```
+or
+```bash
+yakut sub 1003:uavcan.primitive.scalar.Bit.1.0 --with-metadata
```

Comment:
btw you can also say `y` if you don't feel like typing `akut`.

Reply:
Thank you for the hint. I'll keep in mind :bow: :wink: